video_xblock/static/js/context.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Get python context and fill in js global variables with it
+ */
+window.videoPlayerId = '{{ video_player_id }}';
+window.playerStateObj = JSON.parse('{{ player_state }}');
```

Comment:
This context was moved to another file because we need to override it in   js tests